data/ka'het/ka'het.txt

```diff
@@ -73,6 +73,455 @@ phrase "ka'mar"
+	word
+		": online. Secondary systems check: "
+	word
+		"solar wind harvester: online and active"
+		"energy storage: available"
+		"support coolers: online and active"
+		"shield restorers: online and active"
+		"long range antennae: available"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Current objective: systems "
+	word
+		"patrol"
+		"garrison"
+		"vigilance"
+	word
+		". Threats identified: none. Estimated progress: "
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		"%."
+
+phrase "friendly ka'het"
+	word
+		"Het status: "
+	word
+		"operational"
+		"active"
+		"alive"
+		"functioning"
+		"awake"
+	word
+		". Expected lifetime: "
+	word
+		""
+		"1"
+		"2"
+		"3"
+		"3"
+		"3"
+		"4"
+		"4"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years."
+
+phrase "friendly ka'het"
+	word
+		"Current"
+		"Selected"
+		"Chosen"
+		"Main"
+		"Primary"
+	word
+		" target: <ship>. Government: "
+	word
+		"unknown"
+		"unrecognised"
+		"no data"
+		"unidentified"
+	word
+		". Attitude: "
+	word
+		"friendly"
+		"neutral"
+		"peaceful"
+		"positive"
+		"acceptable"
+		"safe"
+		"tolerable"
+		"adequate"
+	word
+		". No necessary action."
+
+phrase "friendly ka'het"
+	word
+		"Connection with the vessel <ship>: "
+	word
+		"established"
+		"stable"
+		"reliable"
+		"strong"
+	word
+		". Estimated signal strength: "
+	word
+		"8"
+		"9"
+		"9"
+		"9"
+	phrase
+		"digit"
+	word
+		"%."
+
+phrase "friendly ka'het"
+	word
+		"Failed connection with "
+	word
+		"planet Osaeli"
+		"planet Alix"
+		"planet Retilie"
+		"planet Veliante"
+		"planet Eavine"
+		"planet Celeatis"
+		"planet Nona"
+		"planet Palarei"
+
+		# Most of the stations mentioned here don't exist anymore at the time of the player, they were destroyed by the Ka'het during the last 200,000 years
+		"station Ceilia"
+		"station Aerite"
+		"station Letis"
+		"station Maiar"
+		"station Lateria"
+		"station Gertima"
+		"station Santari"
+		"station Antamesa"
+		"construction site Delia"
+	word
+		". Reason: "
+	word
+		"no response"
+		"unknown"
+		"missing response"
+	word
+		". Sending report to station "
+	word
+		"Parii"
+		"Seli"
+	word
+		"."
+
+phrase "friendly ka'het"
+	word
+		"Established connection with station "
+	word
+		"Parii"
+		"Seli"
+	word
+		". Signal strength: "
+	word
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+	phrase
+		"digit"
+	word
+		"%. No new data received. Last data received "
+	word
+		""
+		"1"
+		"2"
+	phrase
+		"digit"
+	phrase
+		"digit"
+	word
+		" years, "
+	word
+		""
+		"1 month"
+		"2 months"
+		"3 months"
+		"4 months"
+		"5 months"
+		"6 months"
+		"7 months"
+		"8 months"
+		"9 months"
+		"10 months"
+		"11 months"
+	word
+		", "
+	word
+		"1"
+		"2"
+	phrase
+		"digit"
+	word
+		" days ago. "
+	word
+		"Continuing"
+		"Returning to"
+	word
+		" primary task."
+
+phrase "friendly ka'het"
+	word
+		"Current objective: systems "
+	word
+		"patrol"
+		"garrison"
+		"vigilance"
+	word
+		". Analyzing system <system>. "
```

Comment:
Does this work?

Reply:
Yes, I don't know if I shared screenshots with this one on github, but when I added this I tested it and it worked